src/Aquifer.API/Endpoints/Resources/Content/Get/Endpoint.cs

```diff
@@ -109,20 +109,6 @@ public override async Task HandleAsync(Request request, CancellationToken ct)
             ThrowError("Data integrity issue, no resource content version found.");
         }
 
-        if (relevantContentVersion.IsDraft &&
```

Comment:
Should this be restored?

Reply:
I think yes we do actually want that back since it updates from PendingReview to CompanyReview. Probably need to update that other endpoint to allow it as well.

Reply:
Which other endpoint?